src/uu/chcon/src/fts.rs

```diff
@@ -16,9 +16,9 @@ use crate::os_str_to_c_string;
 
 #[derive(Debug)]
 pub(crate) struct FTS {
-    fts: ptr::NonNull<fts_sys::FTS>,
```

Comment:
no a fan of this one

Reply:
@sylvestre you had a few `NonNull` directly used - i updated those